.gitignore

```diff
@@ -9,4 +9,5 @@ yarn.lock
 /dist
 /.idea
 *.iml
-/tmp
\ No newline at end of file
+/tmp
+key
```

Comment:
what's key ?

Reply:
if you want to keep a key file in the directory 😁

Reply:
instead of stuffing it in the CLI all the time... quite a hassle to go to metamask to copy each time